libr/anal/esil.c

```diff
@@ -150,13 +163,20 @@ R_API bool r_anal_esil_set_pc(RAnalEsil *esil, ut64 addr) {
 	return false;
 }
 
+static int esil_ops_free_cb(void *user, const char *k, const char *v) {
+	RAnalEsilOp *op = (RAnalEsilOp *)(size_t)sdb_atoi (v);
```

Comment:
wat, this is unused code, just return true and drop this line

Reply:
this code is used, and passed to sdb_foreach in r_anal_esil_free